.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+- repo: git://github.com/vintasoftware/pre-commit-hooks
```

Comment:
Add instructions about how to use pre-commits. See: https://github.com/vintasoftware/vinta-site/pull/85/commits/835c67767e8c7741346efd3adb095df98977fda9